components/eam/src/dynamics/se/stepon.F90

```diff
@@ -543,11 +547,13 @@ subroutine stepon_run3(dtime, cam_out, phys_state, dyn_in, dyn_out)
      endif   
 
    endif   
-
+ 
```

Comment:
These whitespace changes are not needed, please revert to clean up the diffs.

Reply:
cleaned